package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "human-connection",
-  "version": "0.2.2",
+  "version": "0.3.0",
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-31T17:25:53Z" title="Friday, January 31st 2020, 6:25:53 pm +01:00">Jan 31, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,6 +1,6 @@
 {
   "name": "human-connection",
-  "version": "0.2.2",
+  "version": "0.3.0",
```


Why the major commit?